sdk/sdkclients/sdkprojectsclient/client.go

```diff
@@ -86,8 +88,9 @@ func (c *client) GetByID(ctx context.Context, pid sdktypes.ProjectID) (sdktypes.
 
 	project, err := sdktypes.StrictProjectFromProto(resp.Msg.Project)
 	if err != nil {
-		// TODO: "errors.Is(err, sdkerrors.ErrInvalidArgument)" doesn't work.
-		if err.Error() == "invalid argument: zero object" {
+		// FIXME: ENG-626: why we check and override errInvalid for project only?
+		var errInvalid sdkerrors.ErrInvalidArgument
+		if err.Error() == "zero object" && errors.As(err, &errInvalid) {
```

Comment:
i think you can use errors.Is?

Reply:
also don't check strings, just make a new error type if you need to.

Reply:
1. InvalidArgument isn't an instance, but a struct. so, I think errors.Is won't work here 
2. it's not my code, so I just left it how it was, including the check of specific string. and added this FIXME: ENG-626 to understand this better and fix in all relevant places